lib/page/questions/uimodel/questions_ui_model.dart

```diff
@@ -0,0 +1,15 @@
+import 'package:equatable/equatable.dart';
+import 'package:survey/model/question_model.dart';
+
+class QuestionsUiModel extends Equatable {
+  final String surveyId;
+  final List<QuestionModel> questions;
```

Comment:
An UiModel should not contain a generic Model. Either using all UiModels or all Models.